api.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const express = require('express');
+
+const cors = require('cors');
+const app = express();
+const io = require('socket.io-client');
+const socket = io.connect('http://localhost:3001');
+
+app.use(express.json());
+app.use(cors());
+
+
+
+
+app.post('/delivery/:vendor/:orderId', (req, res, next) => {
+  const payload = {
+    vendor: req.params.vendor,
+    orderId: req.params.orderId,
+  };
+  console.log('delivery', payload);
+  if (!(payload.vendor === 'flower' || payload.vendor === 'candy')) {

```

Comment:
change 'flower' and 'candy' to 'flower-shop' and 'candy-shop', because that what your referring to in the queue-servers, which then talks to the vendors.